<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="tests.rest.TC011_PutIncident2">
        <methods>
          <include name="putIncident" invocation-numbers="0"/>
          <include name="setValues"/>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="afterSuite"/>
        </methods>
      </class> <!-- tests.rest.TC011_PutIncident2 -->
      <class name="tests.rest.TC011_delete3">
        <methods>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="setValues"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="afterSuite"/>
          <include name="deleteIncident"/>
        </methods>
      </class> <!-- tests.rest.TC011_delete3 -->
      <class name="tests.rest.TC001_CreateIncident">
        <methods>
          <include name="setValues"/>
          <include name="beforeClass"/>
          <include name="createIncident" invocation-numbers="0"/>
          <include name="beforeMethod"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="afterSuite"/>
        </methods>
      </class> <!-- tests.rest.TC001_CreateIncident -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
